packages/shared/src/lib/log.ts

```diff
@@ -149,9 +149,19 @@ export enum LogEvent {
   ScheduleStreakReminder = 'schedule streak reminder',
   // 404 page
   View404Page = '404 page',
-  // source
+  // Source Actions - start
```

Comment:
Just referencing this comment here from previous PR https://github.com/dailydotdev/apps/pull/3360#discussion_r1686342509 from @capJavert 

> Just verify with Ido if this is ok so it can be added to taxonomy sheet (also with other events related changes)

Reply:
What do you think about this? @idoshamun

Reply:
In the taxonomy we have these events already and they follow our legacy convention which is event_name and target_type. It already exists in the code base if you look for it.
We have `block` and `unblock` as event_name and `source` and `tag` as the `target_type`. Let's not change it please and keep logging as it is
<img width="1583" alt="image" src="https://github.com/user-attachments/assets/3f6ba36b-cd53-476f-b0c7-2557add7a287">

Reply:
Currently we are firing `follow` events when user `Unblock` the source, I think this is confusing and is not implemented as the table you mentioned. 
You can check yourself at `packages/shared/src/hooks/useTagAndSource.ts` in `line 209`. Should I keep this? @idoshamun @nensidosari 

And if yes how the event names changes with the new experiment?

Reply:
First, do u have access to view this doc https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit?gid=0#gid=0?
<img width="1567" alt="Screenshot 2024-07-23 at 2 15 01 PM" src="https://github.com/user-attachments/assets/cefffe7b-7dd0-4db9-80ae-5b9d2dfcce7c">

<img width="1572" alt="Screenshot 2024-07-23 at 2 14 28 PM" src="https://github.com/user-attachments/assets/ad7d395d-c9be-4634-ae63-666bf525a9f4">

I think these are the events that refer to follow/block or what we're naming them after block/unblock. 
What Ido is mentioning is that even with the new experiment where we are changing the naming, let's not change events names.

So for this PR we don't want any changes in event names, nor in names for buttons. We can see at the experiment PR if we want to do any changes or add any new events if missing. 🙏

Reply:
> Currently we are firing follow events when user Unblock the source, I think this is confusing and is not implemented as the table you mentioned.
You can check yourself at packages/shared/src/hooks/useTagAndSource.ts in line 209. Should I keep this? @idoshamun @nensidosari

Oh that's not good at all :triggered:

Reply:
@ilasw let's fix it then as suggested. but please stick to the taxonomy definitions

Reply:
For this PR events are ok, updated even in the new one https://github.com/dailydotdev/apps/pull/3385

Reply:
I'm not sure why are we adding more events to the enum. I think there's a confusion here. Please note that in the taxonomy we separate event_name and target_type and here you merge them to one. It's a little tricky so happy to guide you through if needed